staff/francisco-fernandez/pre-course/bingo/project3_v1_pro.js

```diff
@@ -0,0 +1,187 @@
+
+/*Tareas a realizar:
+- Llevar un control de qué números han salido y si se repite sacar otro número al azar OK!!
+- Hacer que los cartones se generen de manera automática y sean siempre diferentes. OK!!
+- Que cheque cada vez que sale un número que la línea no sean todo x par cantar línea OK!!
+- Cuando se muestra el cartón al final de cada turno hay que mostrar el número que se ha encontrado OK!!
+- Hay que llevar la cuenta de los turnos que lleva el jugador OK!!
+- Comprobar que el contador de turnos esté bien OK!!
+- Pendiente ver si quiere volver a jugar OK!!
+- Controlar el boton cancelar del prompt
+
+TAREAS PRO:
+-Cuando se muestre la carta, se preguntará al usuario si realmente quiere ese cartón o generar otro, si realmente
+ quiere ese cartón, deberá responder "yes" para proceder OK!!
+- Establece un sistema de puntos, en cuantos más turnos se complete el cartón, menos puntos (el sistema de puntos
+- intégralo como quieras), por el contrario, a menos turnos, más puntos. OK!!
+- Antes de empezar el juego, muestra el sistema de puntos al usuario. OK!!
+- Ranking de usuarios (ordenado por puntos).OK!! 
+*/
+
+/* creamos una variable global bombo, con el bombo generaremos los cartones y los numeros del juego
+es una variable global para que podamos controlar que no se repitan números*/
+var bombo = bombo = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30];
```

Comment:
I think it's better to write  only var bombo = [....] , not var bombo = bombo = [...];

Reply:
I´ll check it